pkgs/spectest/src/containers/state.zig

```diff
@@ -4,6 +4,8 @@ const configs = @import("@zeam/configs");
 const types = @import("@zeam/types");
 const ssz = @import("ssz");
 const params = @import("@zeam/params");
+const state_transition = @import("@zeam/state-transition");
```

Comment:
Can we follow the leanSpec style to migrate utils function into `BeamState` member functions like previous test. It may wait other related the spec test merged.

Reply:
you're referring to `process_slot` and other functions currently imported from `@zeam/state_transition`?

Yeah, I can do that